hazelcast/src/main/java/com/hazelcast/cp/CPMap.java

```diff
@@ -40,6 +40,7 @@ public interface CPMap<K, V> extends DistributedObject {
      * @param value non-null value of the entry
      * @return null if {@code key} had no previous mapping, otherwise the previous value associated with {@code key}
      * @throws NullPointerException when {@code key} or {@code value} is null
+     * @throws com.hazelcast.core.HazelcastException when an issue arose while performing the {@code put}
```

Comment:
These should either be removed or one should also be added for the CAS. This exception is already thrown but not documented in other proxies.